runtime/tr.source/trj9/optimizer/J9Optimizer.cpp

```diff
@@ -665,6 +665,7 @@ static const OptimizationStrategy cheapWarmStrategyOpts[] =
    { OMR::blockShuffling                                                             },
    { OMR::idiomRecognition,                          OMR::IfLoopsAndNotProfiling     },
```

Comment:
Is this `idiomRecognition` intended to be duplicated on s390 or should it have been deleted from the #ifdef block?

Reply:
This shouldn't be duplicated it should be pulled out of the ifdef

Reply:
Good catch @DanHeidinga. It should **not** be duplicated.